package.json

```diff
@@ -7,6 +7,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "framer-motion": "^2.2.0",
+    "prop-types": "^15.7.2",
```

Comment:
do we really need this?

Reply:
I am using framer motion library cuz It has a bunch of cool animations and transitions. 
I am using prop-types to validate the props I pass to the card component. If I do not validate the props ,the esLint will always throw warnings! I am really not sure about it! we can definitely talk more about that in our daily stand-up.